public/_data.json

```diff
@@ -1,13 +1,13 @@
 {
     "current": {
       "title": "",
-      "datetime": "Thursday April 27th, 2017 7pm",
-      "date": "4/27/2017 19:00:00",
+      "datetime": "Thursday May 25th, 2017 7pm",
+      "date": "5/25/2017 19:00:00",
       "host": {
-        "address": "2415 Michigan Ave, Santa Monica, CA",
-        "name": "Headspace",
-        "misc": "Street parking nearby, and next to the 26th St/Bergamot station 🚃",
-        "url": "https://www.headspace.com/join-us"
+        "address": "2301 Rosecrans Ave, El Segundo, CA 90245",
```

Comment:
This is not the correct location,

`3211 Olympic Blvd, Santa Monica, CA 90404` is